sdk/mixedreality/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_version.py

```diff
@@ -0,0 +1,10 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+
+# matches SEMVER
+VERSION = "0.0.1"
```

Comment:
If this is the first beta release the version should probably match something like `1.0.0b1`